modules/programs/pywal.nix

```diff
@@ -0,0 +1,78 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.pywal;
+
+in {
+  options = { programs.pywal = { enable = mkEnableOption "pywal"; }; };
+
+  config = mkIf cfg.enable {
+
+    home.packages = [ pkgs.pywal ];
+
+    programs.zsh.initExtra = ''
+      # Import colorscheme from 'wal' asynchronously
+      # &   # Run the process in the background.
+      # ( ) # Hide shell job control messages.
+      (cat ~/.cache/wal/sequences &)
```

Comment:
I think you want `${config.xdg.cacheHome}` here (and just below) as well.

Reply:
yes, fixed